packages/components/card/src/card.visualroute.jsx

```diff
@@ -100,6 +100,27 @@ export const component = () => (
       </WrappedCard>
     </Spec>
 
+    <Spec label="Type - Flat, Theme - Dark, Disabled: true">
+      <WrappedCard type="flat" theme="dark" insetScale="s" isDisabled>
+        <Text.Body>{text}</Text.Body>
+      </WrappedCard>
+    </Spec>
+    <Spec label="Type - Flat, Theme - Light, Disabled: true">
+      <WrappedCard type="flat" theme="light" insetScale="s" isDisabled>
+        <Text.Body>{text}</Text.Body>
+      </WrappedCard>
+    </Spec>
+    <Spec label="Type - Raised, Theme - Dark, Disabled: true">
+      <WrappedCard type="raised" theme="dark" insetScale="s">
+        <Text.Body>{text}</Text.Body>
+      </WrappedCard>
+    </Spec>
+    <Spec label="Type - Raised, Theme - Light, Disabled: true">
+      <WrappedCard type="raised" theme="light" insetScale="s">
+        <Text.Body>{text}</Text.Body>
+      </WrappedCard>
```

Comment:
The label should be "Disabled: false" for these 2

Reply:
Thanks for this, great find! Addressed in e4fac58f908e6949718accc8a0a27b188155c797